source/scenes/level/PPLevelSelectScene.cpp

```diff
@@ -15,6 +15,11 @@ bool LevelSelectScene::init(const asset_t &assets) {
     return true;
 }
 
+string LevelSelectScene::getLevel() {
+    string spacer = "-";
+    return _worldName + spacer + _levelNum;
```

Comment:
Not worth it to change this but in future PRs you could've just done _worldName + "-" + _levelNum

Reply:
That has caused issues in the past. Probably fine in this case bc it's output, but C++ gets angy when you do ( _worldName + "-" + _levelNum) as function input

Reply:
Weird. leave it then but I definitely haven't had that issue when doing the texture get for animations